test/e2e/test_cluster_configs/custom_hyperkube.json

```diff
@@ -0,0 +1,46 @@
+{
+    "options": {
+        "allowedOrchestratorVersions": [
+            "1.16"
+        ]
+    },
+    "apiModel": {
+        "apiVersion": "vlabs",
+        "properties": {
+            "orchestratorProfile": {
+                "orchestratorType": "Kubernetes",
```

Comment:
I wonder if the right pattern is to include `"orchestratorVersion": "1.16.0"` here to ensure that any foo that operates on < or > per patch version does the right thing?

Reply:
SGTM

Reply:
The best I can do is setting "orchestratorRelease": "1.16" due to the following error

```
02:44:19  2020/07/09 21:44:19 Output:Error: loading API model in generateCmd: error parsing the api model: the following OrchestratorProfile configuration is not supported: OrchestratorType: "Kubernetes", OrchestratorRelease: "", OrchestratorVersion: "1.16.0". Please use one of the following versions: [1.6.9 1.15.11 1.15.12 1.16.11 1.16.12 1.17.7 1.17.8 1.18.4 1.18.5 1.19.0-alpha.1 1.19.0-alpha.2 1.19.0-alpha.3 1.19.0-beta.0 1.19.0-beta.1 1.19.0-beta.2]

```

Reply:
Yep, I think you're right!

Reply:
So let's roll that back and then lgtm, thanks so much!